cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -110,6 +122,18 @@ void DrawableSubmap::Update(
     // for this submap.
     UpdateTransform();
   }
+  visibility_->setName(QStringLiteral("%1-%2.%3")
```

Comment:
Can we get one fold per trajectory? 

If you run multi trajectory in parallel (which does not yet work, but will eventually), it would be better to have the submaps accessible categorized by trajectory.